flow-server/src/main/java/com/vaadin/flow/server/frontend/installer/NodeInstaller.java

```diff
@@ -538,10 +544,83 @@ private void downloadFileIfMissing(URI downloadUrl, File destination,
                     try {
                         Thread.sleep(DOWNLOAD_ATTEMPT_DELAY * 1000);
                     } catch (InterruptedException e1) {
+                        Thread.currentThread().interrupt();
+                    }
+                } catch (VerificationException ve) {
+                    getLogger().warn("SHA256 verification failed.");
```

Comment:
Since Maven does not show the logger category, it would be good to provide a more specific message
```suggestion
                    getLogger().warn("SHA256 verification of downloaded node bundle failed.");
```

Reply:
used archive instead of bundle, but else updated.